libs/containers/Member.lua

```diff
@@ -464,6 +466,60 @@ function Member:unban(reason)
 	return self._parent:unbanUser(self._user, reason)
 end
 
+function Member:_timeout(val)
+	local data, err = self.client._api:modifyGuildMember(self._parent._id, self.id, {communication_disabled_until = val or json.null})
+	if data then
+		self._communication_disabled_until = val ~= json.null and val or nil
+		return true
+	else
+		return false, err
+	end
+end
+
+--[=[
+@m timeoutFor
+@t http
+@p duration Time/number
+@r boolean
+@d Timeouts the member in its Guild.
+`duration` is either a Time object or a number of seconds representing the duration the timeout stays for.
```

Comment:
Sets a timeout for a guild member. `duration` is either `Time` object or a `number` of seconds representing how long the timeout lasts. To set an expiration date, use `timeoutUntil` instead.